neo/Network/RPC/RpcServer.cs

```diff
@@ -314,6 +314,10 @@ private JObject Process(string method, JArray _params)
                         json["unverified"] = new JArray(unverifiedTransactions.Select(p => (JObject) p.Hash.ToString()));
                         return json;
                     }
+                case "getmempoolnextblockinfo":
+                    {
+                        return Blockchain.Singleton.MemPool.GetMemPoolInfoForNextBlock();
```

Comment:
I think that is better to move here the logic added to `MemoryPool` class, because is json related, and in this class looks weird

Reply:
I agree, you are right Shargon.
Maybe return an array and convert to JSON here.
The logic there is easier for UT and makes some sense.

Reply:
@shargon, take a look now, I changed to Fixed8 and improved the test.
Thanks for this suggestion, it is better and safer like this.
Fell free to change anything or suggest anything else you want.